bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
+    "dist/*.nojson*",
+    "dist/*.named-amd*",
```

Comment:
Maybe, but if so we're not alone :) Ember-CLI [use this approach](http://ember-cli.com/user-guide/#javascript-assets). Also, looking at [main](https://github.com/bower/spec/blob/master/json.md#main) it does allow other files.

Another option would be to make the named AMD default.